### DISCUSSION TITLE
Vocabustudy flyer

### DISCUSSION THREAD

**Discussion by IMGROOT2** [For-0/vocabustudy#221] [General]
Alr, so the dimensions of the flyer you gave me weren't going to work on a sheet of paper, so I recreated it: https://docs.google.com/drawings/d/1PS3ClywwzvrwKHpPiKZOejAV9UqsnSiQcZjnB7D4BRY/edit?usp=sharing. 

But of course, my printer thought it would be a good idea to mistake brown for blue/purple, so I can't print lol.

Can someone else get this done?

**Comment by grimsteel** [comment]
The new flyer looks good, I will try to print it if I can... 
For now we can draw one

**Comment by PrakmO** [comment]
I only have view access - Add vocabustudy.org

**Reply by IMGROOT2** [reply]
i'll add your student email as an editor. 

adding vocabustudy.org now

**Reply by IMGROOT2** [reply]
done
